### PR TITLE
feat: added Ui pages

### PR DESCRIPTION
**Pull request by AnjaliDeshwani**
- Added UI Pages for chrome extension
- Added Google search
- Added context and reducer to set name

### CODE REVIEW

File: src/components/Background/Backgound.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.background-image {
+  z-index: 1;
```
Comment (znjs) [inline src/components/Background/Backgound.css:2]: `z-index` wont work without a `position` property

File: src/components/Background/Backgound.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  background: url("https://images.unsplash.com/photo-1610295388717-7c72886bd20a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80");
+  background-size: cover;
+  background-position: center;
+}
+
+.background-overlay {
+  z-index: 2;
```
Comment (znjs) [inline src/components/Background/Backgound.css:9]: `z-index` wont work without a `position` property

**Review by znjs — APPROVED** [review]
- Clean code 
- Good naming conventions used
- Good folder Structure 
- **extensions are generally SPA, and routing shouldn't be used**
- Can add `onKeyUp` event on name input so that if the user clicks on Enter key input can be taken

File: src/components/Background/Backgound.css (hunk 1 above)
Comment (AnjaliDeshwani) [reply src/components/Background/Backgound.css:2]: @znjs thanks for reviewing. I have directly added position utility class from my component library, it's visible in html structure.

**Comment by AnjaliDeshwani** [comment]
> * **extensions are generally SPA, and routing shouldn't be used**
> * Can add `onKeyUp` event on name input so that if the user clicks on Enter key input can be taken

@znjs Thank you for pointing this out. I have removed routing now.

File: src/components/GoogleSearch/GoogleSearch.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    <div className="search-section">
+      <form
+        method="get"
+        action="https://www.google.com/search"
+        className="search-form"
+      >
+        <i className="fa-solid fa-magnifying-glass search-icon"></i>
+        <input
+          type="text"
+          name="q"
```
Comment (shraddha-amber) [inline src/components/GoogleSearch/GoogleSearch.jsx:12]: you could give some more meaningful name to this input field

File: src/pages/Landing/Landing.jsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  const nameUpdateHandler = () => {
+    dataDispatch({ type: ACTION_TYPE.ADD_USER, payload: { userName: name } });
+  };
+  return (
+    <>
+      <Background />
+      <div className="main-section p-abs top-btm-lft-rgt fx-col fx-ai-center fx-jc-center">
+        <div className="prompt  fx-col fx-ai-center fx-jc-center">
+          <p className="question-text">Hello, What's your name ?</p>
+          <input
```
Comment (shraddha-amber) [inline src/pages/Landing/Landing.jsx:22]: if you put this input field inside a form, you wont have to explicitly check for enter key event

**Review by shraddha-amber — APPROVED** [review]
Clean and well-modularised code!
UI looks pretty good too!

File: src/components/Background/Backgound.css [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  background-color: rgba(0, 0, 0, 0.3);
```
Comment (Nikhil-Nirmal) [inline src/components/Background/Backgound.css:10]: Use CSS variables.

**Review by Nikhil-Nirmal — APPROVED** [review]
Everything Else looks good.

File: src/components/GoogleSearch/GoogleSearch.jsx (hunk 3 above)
Comment (AnjaliDeshwani) [reply src/components/GoogleSearch/GoogleSearch.jsx:12]: Google uses the "q" querystring variable as it's search-query to form the URL, so I have to give this name only.

File: src/pages/Landing/Landing.jsx (hunk 4 above)
Comment (AnjaliDeshwani) [reply src/pages/Landing/Landing.jsx:22]: Thanks @shraddha-1402 for suggesting this change, I have updated it in my code.

File: src/components/Background/Backgound.css (hunk 5 above)
Comment (AnjaliDeshwani) [reply src/components/Background/Backgound.css:10]: Thanks @Nikhil-Nirmal for reviewing it. As I am not planning to use this background color again, that's why I have not used variable.

**Pull request merged**